bspump/kafka/source.py

```diff
@@ -151,17 +155,15 @@ async def main(self):
 						self.LastRefreshTopicsTime = current_time
 						break
 
-					m = c.poll(0.2)
+					m = await self.ProactorService.execute(c.poll, 0.5)  # Increased poll interval
```

Comment:
@PremyslCerny Sorry for the previous comment - this is what I am talking about.
Why did you increased the poll interval?
I suggest to at least make it a global variable and add a comment about why this value is used.
Or make it configurable so we can easily test different values and find the optimal parameterization.

Reply:
@PremyslCerny 

```python
m = await self.ProactorService.execute(c.poll, self.PollInterval)
```

Reply:
I added comments